src/Xamarin.Android.Build.Tasks/Tasks/GeneratePackageManagerJava.cs

```diff
@@ -160,8 +160,9 @@ static internal NativeAssemblerTargetProvider GetAssemblyTargetProvider (string
 		static readonly string[] defaultLogLevel = {"MONO_LOG_LEVEL", "info"};
 		static readonly string[] defaultMonoDebug = {"MONO_DEBUG", "gen-compact-seq-points"};
 		static readonly string[] defaultHttpMessageHandler = {"XA_HTTP_CLIENT_HANDLER_TYPE", "System.Net.Http.HttpClientHandler, System.Net.Http"};
+#if !NETCOREAPP
```

Comment:
We don't actually multitarget `Xamarin.Android.Build.Tasks.csproj`, it is just a `netstandard2.0` project right now...

The way to do this *now* would be to add a new property like:

https://github.com/xamarin/xamarin-android/blob/286683224ea9c38be3028dd03bf47b73f4da45f4/src/Xamarin.Android.Build.Tasks/Tasks/GetAotArguments.cs#L53

And pass in `UsingAndroidNETSdk="$(UsingAndroidNETSdk)"`.